doc/classes/JSONRPC.xml

```diff
@@ -49,16 +61,24 @@
 			<argument index="2" name="id" type="Variant" default="null">
 			</argument>
 			<description>
+			Creates a response which indicates a previous reply has failed in some way.
+			[code]The error code corresponding to what kind of error this is. See the ErrorCode constants.[/code]
+			[code]A custom message about this error.[/code]
+			[code]The request this error is a response to.[/code]
```

Comment:
I guess this was meant to be:
`[code]name[/code] The error code corresponding to what kind of error this is. See the ErrorCode constants.`
etc.?